orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -202,7 +202,7 @@ def OnChange(changeType, level, resource) -> None:  # noqa: ARG001
 
     if changeType == orthanc.ChangeType.STABLE_STUDY and ShouldAutoRoute():
         print("Stable study: %s" % resource)  # noqa: T201
-        SendViaStow(resource)
```

Comment:
Any reason why you touched this? This will conflict with #268 🥲

Reply:
I see you've now changed your code to match?

Reply:
No we made the change already before, which is why it's going to cause a conflict now 🙃 
I don't think this is relevant to the parquet files exporting, so shouldn't be changed here (ideally we would have done this in #226 but I guess we forgot)

Reply:
Seems like an easy confict to resolve, not sure why its a problem?